services/registration/service/registration_service.go

```diff
@@ -14,6 +14,8 @@ var validate *validator.Validate
 
 var db database.Database
 
+const RegistrationAlreadyExists = "Registration already exists."
```

Comment:
I don't the love that fact that I'm using strings to pass around errors. It might be a better idea to create a separate class for errors like this in the common package. Thoughts?

Reply:
I think this is fine. The error types in the common dir seem to be for controller level errors, so I think it's best to keep things consistent (ie. using strings for service level errors, and err types for controller level errors)

Reply:
Yeah, that makes sense.